rpcs3/Emu/Io/usio.cpp

```diff
@@ -79,6 +79,10 @@ usb_device_usio::usb_device_usio(const std::array<u8, 7>& location)
 			.wMaxPacketSize   = 0x0008,
 			.bInterval        = 16}));
 
+	test_sw_on   = false;
```

Comment:
Just as a side note because I was confused on something you said on the other PR. Initializing the values here and in the header is functionally the same. It's actually better to initialize the values in the header as it will set the values in a representation of the class at compile time(vs initializing it in the constructor when the object is constructed).

Reply:
> Just as a side note because I was confused on something you said on the other PR. Initializing the values here and in the header is functionally the same. It's actually better to initialize the values in the header as it will set the values in a representation of the class at compile time(vs initializing it in the constructor when the object is constructed).

I meant the those values should always be reset whenever the game is restarted (hence reconstruct the usb_device_usio).

Reply:
Yes and they are if you set them in the class definition.

Reply:
makes sense.